core/src/test/resources/org/fao/geonet/api/Messages_fre.properties

```
@@ -65,6 +69,18 @@ register_email_message=Cher utilisateur,\n\
   \n\
   Salutations,\n\
   L'\u00E9quipe %s.
+register_email_group_message=Cher utilisateur,\n\
+  Votre compte a \u00E9t\u00E9 cr\u00E9\u00E9 avec succ\u00E9s pour %s.\n\
+  * Nom d'utilisateur : %s\n\
+  * Mot de passe : %s\n\
+  * Profil : %s\n\
+  \n\
+  Vous avez demand\u00E9 un profil %s en groupe %s, vous serez contact\u00E9 par notre \u00E9quipe prochainement.\n\
```

```suggestion
Nous avons envoyé votre demande de %s à l'administrateur du groupe %s. Vous serez contacté rapidement.\n\
```

The use of `de reviewer` or `d' editor` how do we handle that?